src/applications/ivc-champva/10-7959f-1/10-7959F-1-schema.json

```diff
@@ -0,0 +1,216 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Foreign Medical Program (FMP) Registration Form (10-7959F-1)",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "address": {
```

Comment:
These addresses are in a few other locations as well, is there some way we can inherit them? Not sure if it works that way cause I haven't used the json-schema before. Other than that question, everything looks good!